msteams-platform/tabs/how-to/authentication/auth-silent-aad.md

```diff
@@ -7,6 +7,9 @@ keywords: teams authentication SSO silent AAD tab
 ---
 # Silent authentication
 
+> [!IMPORTANT]
+> All Microsoft support and development for ADAL including security fixes, ends on **June 30, 2022**. You can migrate your app that uses the ADAL.js to use MSAL.js  to continue receiving support. For more information, see [Migrate applications to the Microsoft Authentication Library (MSAL)](/azure/active-directory/develop/msal-migration).
```

Comment:
"You can migrate your app ~~that uses the ADAL.js~~ to use MSAL.js to continue receiving support"

No need to clarifying ADAL.js to MSAL.js migration in this sentence since ADAL is mentioned in the previous sentence.